### ISSUE TITLE
nom and nvd missing on path

### ISSUE THREAD

**Issue by gigamonster256** [ToyVo/nh_darwin#11]
Prior to the changes today, nom did not need to be available on $PATH for nh-darwin to work. However, now it does. This makes nh_darwin invocations fail if the nom binary is not available and no_nom is not specified.

The reason is that nvd and nom are runtime deps but were incorrectly swapped to be build time when package.nix was moved into flake.nix

**Comment by ToyVo** [comment]
Yes I am aware and am working on a fix. Interestingly the behavior is different on Linux vs Darwin. 

The motivation was to switch to crate2nix for faster builds so the dependencies could take advantage of the nix cache.

I would suggest pinning to an earlier Rev temporarily.
